### DISCUSSION TITLE
3.25.0

### DISCUSSION THREAD

**Discussion by zenangst** [zenangst/KeyboardCowboy#558] [Announcements]
<img width="656" alt="Screenshot 2024-10-24 at 23 28 16" src="https://github.com/user-attachments/assets/9d22aaeb-e0a7-42b3-a252-adf1004e3f9b">

> [!IMPORTANT]
> **Heads up, cowboys!** 
> The configuration file has packed its bags and moved from `~/.keyboard-cowboy.json` to the new and fabulous `~/.config/keyboardcowboy/config.json`. Because even config files deserve a glow-up!

- Level up, darling! macOS productivity just got a major glow-up with Workspaces. You’re about to become unstoppable.
- Smart Window Tiling in macOS Sequoia—because who has time for messy windows? Stay sleek.
- Get your focus on with the fabulous App Focus command. Say goodbye to distractions.
- Peek-a-boo! Quick peek at your apps with our sneaky new peek functionality. We see you.
- Fly through apps with quarter & center focus commands. Zoom, zoom, zoom between windows like a boss!
- Repeat after me: Easily replay your last workflow or keyboard event. Repetition has never been this satisfying.
- Messy workspace? Not anymore! Clean it up with the new Smart Hide All Other Apps command. Clutter? Never heard of it.
- Keyboard Cleaner—because keeping your keyboard crumb-free is a vibe.
- Application commands now support ‘wait until launched’—we’re patient, but only just enough.
- Last app command activation is now rock solid. Trust us, you won’t miss a beat.
- Window focus shifts like a pro—because precision matters.
- Macros with iterations? Yes, please. More power, less effort.
- New scanning algorithm—it’s a complete glow-up for relative focus. Navigate like a boss.
- Moving windows between monitors is now smoother than ever. Glide between screens like you own the place.
- Application Triggers now support ‘any Application’—because options are everything.
- Notifications powered by DynamicNotchKit—fancy, right? We thought so too.
- Bug fixes galore—because we can’t stand anything less than perfection.

<hr /><em>This discussion was created from the release <a href='https://github.com/zenangst/KeyboardCowboy/releases/tag/3.25.0'>3.25.0</a>.</em>

**Comment by nonameolsson** [comment]
Awesome!
Is there a place where to read more about each feature? Like
> Level up, darling! macOS productivity just got a major glow-up with Workspaces. You’re about to become unstoppable.

**Reply by zenangst** [reply]
@nonameolsson I pushed some updates to the wiki now.

https://github.com/zenangst/KeyboardCowboy/wiki/Commands#workspaces
